modules/Hoot/managers/layerManager.js

```diff
@@ -310,6 +311,8 @@ export default class Layers {
             let coordsList;
             if ( lyr.bounds && lyr.bounds.includes( ';' ) ) {
                 coordsList = polyStringToCoordsList( lyr.bounds );
+            } else {
+                coordsList = _cloneDeep( [ polyCoords ] );
```

Comment:
what was this fixing?  a single polygon vs multiple?

Reply:
yea the else is for single